### ISSUE TITLE
getShortUrl creates incorrect data on web

### ISSUE THREAD

**Issue by stx** [RodrigoSMarques/flutter_branch_sdk#137]
Working around the fact that initWeb() doesn't work by adding it to the index.html script tag, there's an additional issue with `getShortUrl`:

Running this code on the web and mobile results in two different results:

```
    BranchUniversalObject buo = BranchUniversalObject(
      canonicalIdentifier: 'referral',
      canonicalUrl: 'https://mysite.com',
      publiclyIndex: false,
      locallyIndex: false,
    );

    BranchLinkProperties lp = BranchLinkProperties(
      feature: 'referral',
    )..addControlParam('referring_user_id', 'asdf');

    BranchResponse<dynamic> response = await FlutterBranchSdk.getShortUrl(
      buo: buo,
      linkProperties: lp,
    );
```

Running this code via Flutter web, and visiting the resulting URL on a desktop (and landing on the web URL) results in **referring_user_id** is missing completely and **$canonical_url** is the URL the user was on when the link was created.

Running this code via Flutter mobile, and visiting the resulting URL on a desktop (and landing on the web URL) results **referring_user_id** being correct and **$canonical_url** being set to the value specified.

**Comment by RodrigoSMarques** [comment]
Problem identified. Fix in next version

**Comment by RodrigoSMarques** [comment]
Fixed in version 5.0.0.
If necessary open again.
